<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="processButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>130</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Get</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>751</width>
        <height>56</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Youtube Link (Playlist or Video)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linkLineEdit">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="downloadButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>420</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>751</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>221</y>
        <width>751</width>
        <height>191</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>194</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Youtube Downloader</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopDownloadButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>500</y>
        <width>161</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Downloading</string>
      </property>
     </widget>
     <widget class="QLabel" name="label4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>631</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Label 4</string>
      </property>
     </widget>
     <widget class="QLabel" name="downloadStatusLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>500</y>
        <width>581</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="loaderLabel">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>10</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="resetButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>170</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QLabel" name="downloadLocationLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>530</y>
        <width>761</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>631</width>
        <height>77</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label1">
         <property name="text">
          <string>Label 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label2">
         <property name="text">
          <string>Label 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label3">
         <property name="text">
          <string>Label 3</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
